### PR TITLE
ci: release debian package

### LINKED ISSUE #143: Release Debian package with cargo-deb
https://github.com/mmstick/cargo-deb

### PR DESCRIPTION
**Pull request by Wenzel**
fix #143

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -228,6 +228,39 @@ jobs:
...
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
```
Comment (rageagainsthepc) [inline .github/workflows/ci.yml:243]:
```suggestion
          toolchain: stable
          components: cargo-deb
```

**Comment by rageagainsthepc** [comment]
@Wenzel Where should I see the published debian package?

**Comment by Wenzel** [comment]
So, the goal of the CI is this:
EpeizubXYAccwP9

Separate debian package generation and publication

publication only happens when we push a new release tag
I wanted to get the generated package for every PR, as it comes handy.

I think the artifacts upload for github actions are only visible to repo owner ?
Capture d’écran de 2020-12-18 11-38-07

**Comment by rageagainsthepc** [comment]
No, I can see them as well. I just expected to see them in the release section.

**Pull request merged**